src/HIE/Bios/Environment.hs

```diff
@@ -76,7 +77,11 @@ however, it's not really necessary as
 >   when res (removeDirectoryRecursive cd)
 -}
 getCacheDir :: FilePath -> IO FilePath
-getCacheDir fp = getXdgDirectory XdgCache (cacheDir </> fp)
+getCacheDir fp = do
+  mbEnvCacheDirectory <- lookupEnv "HIE_BIOS_CACHE_DIR"
+  cacheBaseDir <- maybe (getXdgDirectory XdgCache cacheDir) return
+                         mbEnvCacheDirectory
+  return (cacheBaseDir </> fp)
```

Comment:
Documentation for this function please! 
Also, `getCacheDir` is probably a misnomer.

Reply:
Mmm it retrieves the cache directory used by the lib, `getHieBiosCacheDir` would be better or you are thinking in another deficiency?

Reply:
Or maybe a misnomer... Mainly adding documentation for the env variable and so on suffices for me.